support-frontend/assets/pages/digital-subscription-checkout/helpers/paymentProviders.js

```diff
@@ -123,6 +124,9 @@ function showPaymentMethod(
     case 'DirectDebit':
       dispatch(openDirectDebitPopUp());
       break;
+    case 'PayPal':
+      // PayPal is handled differently
```

Comment:
a bit more information about how & why PayPal is different could be useful in this comment